test/files/rules/quotemark.test.ts

```diff
@@ -1,2 +1,4 @@
 var single = 'single';
     var doublee = "married";
+        var single = "'single escaped by doublee'";
+            var doublee = '"doublee escaped by single"';
```

Comment:
can you un-indent these lines please? no need to fix line 2.
also name these variables `singleWithinDouble` and `doubleWithinSingle`